backend/hitas/templates/unconfirmed_maximum_price.jinja

```diff
@@ -0,0 +1,113 @@
+        <br>{{ apartment.address.postal_code }} {{ apartment.address.city|upper }}
+    </h3>
+
+
+    <h4>HITAS-HUONEISTON ENIMMÄISHINTA<br>
+    • ilman yhtiökohtaisia parannuksia ja mahdollista yhtiölainaosuutta</h4>
+
+    <table>
+        {% for ownership in apartment.ownerships %}
+        <tr>
+            <td>Omistaja ja omistusosuus (%)</td>
+            <td>{{ ownership.owner.name}}</td>
+            <td>{{ ownership.percentage }}</td>
+        </tr>
+        {% endfor %}
+    </table>
+
+    <br>
+    <br>
+
+    <table>
+        <tr>
+            <td>Asunto-osakeyhtiö</td>
+            <td>{{ apartment.links.housing_company.display_name }}, Helsinki</td>
+        </tr>
+        <tr>
+            <td>Huoneiston osoite</td>
+            <td>{{ apartment.address.street_address }} {{ apartment.address.stair }} {{ apartment.address.apartment_number }}, {{ apartment.address.postal_code }} {{ apartment.address.city|upper }}</td>
+        </tr>
+        <tr>
+            <td>Alkuperäinen velaton hankintahinta, euroa</td>
+            <td>{{ apartment.prices.debt_free_purchase_price|intcomma }}</td>
+        </tr>
+        <tr>
+            <td>Huoneiston valmistumisajankohta</td>
+            <td>{{ apartment.completion_date|format_date }}</td>
+        </tr>
+        <tr>
+            <td>Huoneiston ensimmäinen kaupantekoajankohta</td>
+            <td>{{ apartment.first_purchase_date|format_date }}</td>
+        </tr>
+        {% if is_surface_area_price_ceiling_maximum %}
+            <tr>
+            {% if prices.construction_price_index.value > prices.market_price_index.value %}
+                <td>Hitas-huoneiston velaton enimmäishinta rakennuskustannusindeksillä laskettuna, euroa</td>
+                <td>{{ prices.construction_price_index.value|intcomma }}</td>
+            {% else %}
+                <td>Hitas-huoneiston velaton enimmäishinta markkinahintaindeksillä laskettuna, euroa</td>
+                <td>{{ prices.market_price_index.value|intcomma }}</td>
+            {% endif %}
+            </tr>
+        {% endif %}
+    </table>
+
+    <br>
+
+    {% if not is_surface_area_price_ceiling_maximum %}
+        <p style="font-size: 9pt">
+        Lisätietokenttään kirjoitetaan, jos laskelmassa on jotain erityistä, mitä osakkaan on syytä tietää.
+        </p>
+    {% endif %}
+
+    <b>Hitas-huoneistonne velaton enimmäishinta on
+        {% if prices.construction_price_index.maximum %}
+            {{ prices.construction_price_index.value|intcomma }} euroa rakennuskustannusindeksillä laskettuna.
+        {% elif prices.market_price_index.maximum %}
+            {{ prices.market_price_index.value|intcomma }} euroa markkinahintaindeksillä laskettuna.
+        {% else %}
+            {{ prices.surface_area_price_ceiling.value|intcomma }} euroa. Velaton enimmäishinta on laskettu voimassaolevan rajahinnan perusteella ({{ apartment.surface_area }} m² * ???? euroa).
```

Comment:
the `????` needs fixing. It's something that would be part of the **confirmed** calculation but not the unconfirmed. So it needs to be fetched separately for this rendering context

Reply:
The `????` part is the index value for _Rajaneliöhinta_, which is not included in the unconfirmed calculations, so we need to get it separately yes. Left this here as a TODO to fix a bit later :)